.github/workflows/QA-1612-test.yml

```diff
@@ -0,0 +1,126 @@
+# This workflow runs the WSM full regression tests, including perf, integration, and
```

Comment:
Should this file be included in the change set? I assume it was for your testing.

Reply:
Again this is for testing only. I'll let you know once this PR is ready for wider review after @marikomedlock and I finished finalizing PR#[21](https://github.com/DataBiosphere/terra-test-runner/pull/21).

Reply:
And this also still needs to be removed.